packages/generator/src/generators/page-generator.ts

```diff
@@ -47,7 +47,8 @@ export class PageGenerator extends Generator<PageGeneratorOptions> {
   }
 
   getModelNamesPath() {
-    const context = this.options.context ? `${this.options.context}/` : ""
+    const kebabCaseContext = camelCaseToKebabCase(this.options.context)
+    const context = this.options.context ? `${kebabCaseContext}/` : ""
     return context + this.options.modelNames
```

Comment:
You also need to kebab case `this.options.modelNames`

Reply:
ah gotcha, created a new PR now. Hopefully that solves it!